lib/dao.js

```diff
@@ -2293,6 +2293,7 @@ DataAccessObject.updateAll = function(where, data, options, cb) {
         if (!(data instanceof Model)) {
           try {
             inst = new Model(data, {applyDefaultValues: false});
+            ctx.data = inst.toObject(true);
```

Comment:
what does `true` mean here?

Reply:
Good question. From the API documentation, it'll convert model instance to Plain Data Object for properties defined in the schema:
```
/**
 * Convert model instance to a plain JSON object.
 * Returns a canonical object representation (no getters and setters).
 *
 * @param {Boolean} onlySchema Restrict properties to dataSource only.  Default is false.  If true, the function returns only properties defined in the schema;  Otherwise it returns all enumerable properties.
```
I followed what was done for create (see https://github.com/strongloop/loopback-datasource-juggler/blob/master/lib/dao.js#L353) for this statement.